### DISCUSSION TITLE
https://www.yunyitang.me/regular-expression-basics/

### DISCUSSION THREAD

**Discussion by giscus** [yunyit/yunyit.github.io#26] [Comments]
# [正则表达式基础知识点](https://www.yunyitang.me/regular-expression-basics/)

认知始于 Excel，不知道终于哪儿。 介绍与应用 属性：用于匹配和操作文本的工具 设计思想：用一种描述性的语言来给字符串定义一个规则，凡是符合规则

https://www.yunyitang.me/regular-expression-basics/

**Comment by jiangyangcreate** [comment]
我在找用法的时候更喜欢思维导图，这是我整理的，可以参考下https://jiangmiemie.com/assets/images/re-16baebe2fd788bda7f341ab9bf45b828.jpg

**Reply by yunyit** [reply]
谢谢分享！上面的图链无效了，不过你的博客好棒，关注了～～

**Reply by jiangyangcreate** [reply]
感谢关注，因为后面我做博客无图化，所以改成这样的动态的思维导图了，供参考https://jiangmiemie.com/docs/%E9%80%86%E5%90%91%E6%B5%8B%E8%AF%95/%E6%95%B0%E6%8D%AE%E6%8F%90%E5%8F%96/#%E6%AD%A3%E5%88%99%E8%A1%A8%E8%BE%BE%E5%BC%8F
